test/e2e/tests/metrics/errors.spec.js

```diff
@@ -114,7 +114,7 @@ async function matchesSnapshot({
   const snapshotData = JSON.parse(rawSnapshotData);
 
   try {
-    assert.deepStrictEqual(data, snapshotData);
+    isEqual(data, snapshotData);
```

Comment:
If the snapshot data does not match, won't `isEqual` won't throw an error, so this test will pass even if it should fail. We probably need this to be `assert(isEqual(data, snapshotData))`

Reply:
Oooooops... sorry, not sure why I removed the assert, it wasn't intentional. I've corrected it in the latest commit. Thanks for the great catch!